tests/test_pauli.py

```diff
@@ -38,7 +34,7 @@ def test_matmul(self, a: Pauli, b: Pauli) -> None:
     @pytest.mark.parametrize(
         ("plane", "s", "t", "clifford", "angle", "choice"),
         itertools.product(
-            graphix.pauli.Plane,
+            Plane,
             (False, True),
             (False, True),
             iter(Clifford),
```

Comment:
I just realized that `iter` is useless here.
```suggestion
            Clifford,
```